addons/ekco/0.27.1/service.tmpl.yaml

```diff
@@ -3,8 +3,10 @@ kind: Service
 metadata:
   name: ekc-operator
 spec:
+  type: NodePort
   selector:
     app: ekc-operator
   ports:
     - protocol: TCP
-      port: 8080
\ No newline at end of file
+      port: 8080
+      nodePort: $EKCO_NODE_PORT
```

Comment:
is there a reason to specify the nodeport, instead of getting it after the fact?

Reply:
if `nodePort` not explicitly specified, the control planes will assign one for you in this range: `30000-32767`. I'd rather we assign a specific port to the EKCO server for consistency and ease of discoveribility.